packages/pds/tests/sync/subscribe-repos.test.ts

```diff
@@ -104,6 +120,22 @@ describe('repo subscribe repos', () => {
     return evts
   }
 
+  const getAllEvents = (userDid: string, frames: Frame[]) => {
+    const types = []
+    for (const frame of frames) {
+      if (frame instanceof MessageFrame) {
+        if (
+          (frame.header.t === "#commit" && (frame.body as CommitEvt).repo === userDid)
+          || (frame.header.t === "#handle" && (frame.body as HandleEvt).did === userDid)
+          || (frame.header.t === "#tombstone" && (frame.body as TombstoneEvt).did === userDid)
```

Comment:
same thing here wrt enumerating all the types

Reply:
though in this situation i don't think there's a better way to extract a did from body without doing this